src/Uno.UI/UI/Xaml/UIElement.Layout.crossruntime.cs

```diff
@@ -255,6 +257,13 @@ private void DoMeasure(Size availableSize)
 #endif
 					{
 						LayoutInformation.SetAvailableSize(this, availableSize);
+
+						if (previousDesiredSize != LayoutInformation.GetDesiredSize(this) &&
+							!IsMeasureDirtyPathDisabled &&
+							this.GetParent() is UIElement parent)
+						{
+							parent.InvalidateMeasure();
```

Comment:
Adding explanations...

```suggestion
							// When the desired size is changing, it is important to make sure to redo the measurement
							// on the parent element. This case usually occurs if the .Measure() is invoked outside
							// the measure+arrange phases.

							parent.InvalidateMeasure();
```